.github/workflows/config.yml

```diff
@@ -300,6 +343,7 @@ jobs:
           cp -r results/upgrades/timeseries/*.csv test/base_results/upgrades/timeseries
           git add test/base_results
           git add docs
+          git add resources/options_lookup.tsv
```

Comment:
Looks like the artifact download correctly places the options_lookup.tsv in the resources folder. So that's good - no copy necessary.